### PR TITLE
Packages upgrade

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

File: src/Filter/Filter.php [hunk 1]
```diff
@@ -17,41 +20,11 @@
...
- * @author  Jefersson Nathan <malukenho@phpse.net>
- * @license MIT
- */
-final class Filter
+interface Filter
 {
     /**
-     * @var string
+     * Receives the docheader content, may already processed by other
+     * filter and apply some changes, them return it.
```
Comment (Ocramius) [inline src/Filter/Filter.php]: then

File: src/Filter/Filter.php [hunk 2]
```diff
@@ -17,41 +20,11 @@
  */
 namespace DocHeader\Filter;
 
-/**
- * @author  Jefersson Nathan <malukenho@phpse.net>
- * @license MIT
- */
-final class Filter
```
Comment (Ocramius) [inline src/Filter/Filter.php:24]: This is a BC break

**Review by Ocramius — CHANGES_REQUESTED** [review]
Patch looks ok-ish, but you really need to run it through `roave/backwards-compatibility-check` first

File: src/Filter/Filter.php (hunk 2 above)
Comment (malukenho) [reply src/Filter/Filter.php:24]: Yep. It will be a major release since I'll get rid of php >=5.5 as well

**Pull request merged**